UnitTests/GitCommandsTests/GitModuleTest.cs

```diff
@@ -572,5 +572,13 @@ public void GetRefsCmd(string expected, bool tags, bool branches, bool noLocks)
         {
             Assert.AreEqual(expected, _gitModule.GetRefsCmd(tags, branches, noLocks).ToString());
         }
+
+        [TestCase(false, @"-c core.safecrlf=false update-index --add --stdin")]
+        [TestCase(true, @"update-index --add --stdin")]
+        public void UpdateIndexCmd_should_handle_core_safecrlf(bool showErrorsWhenStagingFiles, string expected)
+        {
+            var actual = _gitModule.UpdateIndexCmd(showErrorsWhenStagingFiles).ToString();
+            StringAssert.AreEqualIgnoringCase(expected, actual);
```

Comment:
Git commands are case sensitive, so the assertion should be too

Reply:
You're right.